src/oatpp/network/tcp/server/ConnectionProvider.cpp

```diff
@@ -277,10 +277,10 @@ oatpp::v_io_handle ConnectionProvider::instantiateServer(){
 
       if (setsockopt(serverHandle, SOL_SOCKET, SO_REUSEADDR, &yes, sizeof(int)) != 0) {
         OATPP_LOGW("[oatpp::network::tcp::server::ConnectionProvider::instantiateServer()]",
-                   "Warning. Failed to set %s for accepting socket: %s", "SO_REUSEADDR", strerror(errno));
+                   "Warning. Failed to set %s for accepting socket: %s", "SO_REUSEADDR", strerror(errno))
       }
 
-      if (bind(serverHandle, currResult->ai_addr, static_cast<int>(currResult->ai_addrlen)) == 0 &&
+      if (bind(serverHandle, currResult->ai_addr, static_cast<socklen_t>(currResult->ai_addrlen)) == 0 &&
```

Comment:
Let's use a newly introduced v_sock_size

Reply:
merged master that has this